src/lib/webpack/webpack-base-config.js

```diff
@@ -142,7 +142,7 @@ export default function (env) {
 							{
 								loader: 'postcss-loader',
 								options: {
-									sourceMap: isProd,
+									sourceMap: true,
```

Comment:
can't we leave this in the user land whether to use sourceMaps in production or not?

Reply:
This isn't production. This just always _allows_ PostCSS to accept source maps. Whether or not source maps are created is determined by the `(css|sass|stylus|less)-loader`s. 

This can still be overridden in Userland, if wanted, but there'd be no point.

Development sourcemaps don't work with out this.